pipeline_dp/combiners.py

```diff
@@ -192,6 +196,10 @@ def compute_metrics(self, count: AccumulatorType) -> dict:
     def metrics_names(self) -> List[str]:
         return ['count']
 
+    def explain_computation(self) -> str:
```

Comment:
The type annotation should be Callable. ditto for the following types.

Reply:
done